joplin/static/js/admin.js

```diff
@@ -1,12 +1,55 @@
-document.addEventListener('DOMContentLoaded', function(event) {
+// TODO: This a better way
+const anchorLinks = {
+  id_title:
```

Comment:
what if we had a key that was the `baseurl` so then each of the following `id_foo` are much shorter? ex:

```
{
  basurl:  `https://briaguya.github.io/digital-services-style-guide/writing-service-pages/`,
  id_steps: `#steps`,
  ..
}
```